app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.kt

```diff
@@ -132,3 +133,22 @@ fun setProfileImage(imageView: ImageView, profileAvatar: ProfileAvatar?) {
       .into(imageView)
   }
 }
+
+@BindingAdapter("app:expand_rotation_anim")
```

Comment:
This animation has no role in expand/collapse of the list and can be used independently anywhere.
Therefore rename this to:
`app:rotationAnimation`

Also, pass change the `isExpanded` to `isClockwise` which can be used to control clockwise/anticlockwise rotation.
Also, If possible pass angle in the function so that it because completely reusable at different places.

Reply:
Done